### PR TITLE
doc: new quick start doc with more detail and gif

### PR DESCRIPTION
**Pull request by zhongjiajie**
copy paste from https://github.com/bitsondatadev/trino-getting-started/pull/22

and the chinese translate by Google Translate

### CODE REVIEW

**Review by SbloodyS — DISMISSED** [review]
LGTM.

File: docs/docs/en/guide/start/quick-start.md [hunk 1]
```diff
@@ -1,61 +1,118 @@
...
 <figure class="video_container"> 
   <iframe src="https://www.youtube.com/embed/nrF20hpCkug" frameborder="0" allowfullscreen="true"></iframe>
 </figure>
 
-## Administrator User Login
+## Text Tutorial
+
+### Setup Dolphinscheduler
+
+You have to install and start dolphinscheduler first before go ahead. For the beginner, we recommend setup up
```
Comment (EricGao888) [inline docs/docs/en/guide/start/quick-start.md]:
```suggestion
You have to install and start dolphinscheduler first before go ahead. For beginners, we recommend setting up
```

File: docs/docs/en/guide/start/quick-start.md [hunk 2]
```diff
@@ -1,61 +1,118 @@
...
+dolphionscheduler with the official Docker image or starting the standalone server.
```
Comment (EricGao888) [inline docs/docs/en/guide/start/quick-start.md]:
```suggestion
DolphinScheduler with the official Docker image or with the standalone server.
```

File: docs/docs/en/guide/start/quick-start.md [hunk 3]
```diff
@@ -1,61 +1,118 @@
...
+
+* [standalone server](https://dolphinscheduler.apache.org/en-us/docs/3.1.3/guide/installation/standalone)
+* [docker](https://dolphinscheduler.apache.org/en-us/docs/3.1.3/guide/start/docker)
+
+### Build Your First Workflow
+
+You can log in dolphinscheduler with http://localhost:12345/dolphinscheduler/ui the default username/password
```
Comment (EricGao888) [inline docs/docs/en/guide/start/quick-start.md]:
```suggestion
You can log in DolphinScheduler through http://localhost:12345/dolphinscheduler/ui and the default username/password
```

File: docs/docs/en/guide/start/quick-start.md [hunk 4]
```diff
@@ -1,61 +1,118 @@
...
+is `admin/dolphinscheduler123`.
+
+![login](../../../../img/start/login.gif)
+
+#### Create Tenant
+
+Tenant is a concept that cannot be bypassed while using DolphinScheduler, so
```
Comment (EricGao888) [inline docs/docs/en/guide/start/quick-start.md]:
```suggestion
Tenant is an important concept while using DolphinScheduler, so
```

File: docs/docs/en/guide/start/quick-start.md [hunk 5]
```diff
@@ -1,61 +1,118 @@
...
+let's briefly introduce the concept of tenant first.
+
+The account named `admin` logged into DolphinScheduler is called user in dolphinscheduler.
```
Comment (EricGao888) [inline docs/docs/en/guide/start/quick-start.md]: DolphinScheduler maps the `admin` account you use to log into DolphinScheduler to `user`.

File: docs/docs/en/guide/start/quick-start.md [hunk 6]
```diff
@@ -1,61 +1,118 @@
...
+To better control system resources, DolphinScheduler introduce the concept of
```
Comment (EricGao888) [inline docs/docs/en/guide/start/quick-start.md]:
```suggestion
To better control system resources, DolphinScheduler introduces the concept of
```

File: docs/docs/en/guide/start/quick-start.md [hunk 7]
```diff
@@ -1,61 +1,118 @@
...
+
+We can create a tenant in DolphinScheduler `Security -> Tenant Manage` page.
+
+![create-tenant](../../../../img/start/create-tenant.gif)
+
+#### Assign Tenant to User
 
-* Address: http://localhost:12345/dolphinscheduler
-* Username and password: admin/dolphinscheduler123
+As we talked about above in Create Tenant, the user can only run the task unless the user is assigned to a tenant.
```
Comment (EricGao888) [inline docs/docs/en/guide/start/quick-start.md]:
```suggestion
As we have shown above in `Create Tenant` section, the`user` can not run tasks until we assign it with a `tenant`.
```

File: docs/docs/en/guide/start/quick-start.md [hunk 8]
```diff
@@ -1,61 +1,118 @@
...
 
-![create-queue](../../../../img/new_ui/dev/quick-start/create-queue.png)
+After we create a tenant and assign it to a user, we can start creating a
+simple workflow in DolphinScheduler.
 
-## Create a tenant
+#### Create Project
 
-![create-tenant](../../../../img/new_ui/dev/quick-start/create-tenant.png)
+But in DolphinScheduler, all workflow must belong to a project, so we need
```
Comment (EricGao888) [inline docs/docs/en/guide/start/quick-start.md]:
```suggestion
But in DolphinScheduler, all workflows must belong to a project, so we need
```

File: docs/docs/en/guide/start/quick-start.md [hunk 9]
```diff
@@ -1,61 +1,118 @@
...
+![create-workflow](../../../../img/start/create-workflow.gif)
 
-![create-alarmGroup](../../../../img/new_ui/dev/quick-start/create-alarmGroup.png)
+#### Create Tasks
 
-## Create a worker group
+We can use the mouse to drag the task you want to create from the toolbar in the workflow canvas.
+In this case, we create a `Shell` task. Entering the necessary information for the task,
+we just fill the attribute `Node Name` with `Script` to the task for this simple workflow.
+After that, we can click the `Save` button save the task into the workflow. We create another task
```
Comment (EricGao888) [inline docs/docs/en/guide/start/quick-start.md]:
```suggestion
After that, we can click the `Save` button to save the task into the workflow. We create another task
```

File: docs/docs/en/guide/start/quick-start.md [hunk 10]
```diff
@@ -1,61 +1,118 @@
...
-## Create environment
+#### Set Task Dependency
 
-![create-environment](../../../../img/new_ui/dev/quick-start/create-environment.png)
+So we have two different tasks with different names and commands to run in the workflow. The
+only thing missing from the current workflow is task dependency. We can add dependency using
+the mouse to drag the arrow from the upstream task to the downstream and then release the mouse.
+And you can see the link with the arrow between the two tasks is created, from the upstream
+task to the downstream one. Finally, we can click the `Save` button from the top right corner
+to save the workflow, do not forget to fill name of the workflow.
```
Comment (EricGao888) [inline docs/docs/en/guide/start/quick-start.md]:
```suggestion
to save the workflow, do not forget to enter the name of the workflow.
```

File: docs/docs/en/guide/start/quick-start.md [hunk 11]
```diff
@@ -1,61 +1,118 @@
...
-* `Project Management -> Create Project -> Project Name`
+#### View Log
 
-![project](../../../../img/new_ui/dev/quick-start/project.png)
+If you want to view the task log, please click the workflow instance from the workflow instance
+list, then find the task you want to view the log, right-click the mouse and select `View Log`
+from the context dialog, and you can see the detailed log of the task.
 
-* `Click Workflow Definition -> Create Workflow Definition-> Online Workflow Definition`
+You can the task print the `Hello DolphinScheduler` and `Ending...` which is the same as we
```
Comment (EricGao888) [inline docs/docs/en/guide/start/quick-start.md]:
```suggestion
You can see the task printing `Hello DolphinScheduler` and `Ending...` which is the same as we
```

File: docs/docs/en/guide/start/quick-start.md [hunk 12]
```diff
@@ -1,61 +1,118 @@
...
+define in create the task.
```
Comment (EricGao888) [inline docs/docs/en/guide/start/quick-start.md]:
```suggestion
defined when creating the tasks.
```

File: docs/docs/en/guide/start/quick-start.md [hunk 13]
```diff
@@ -1,61 +1,118 @@
...
 
-![workflow-definition](../../../../img/new_ui/dev/quick-start/workflow-definition.png)
-* `Running Workflow Definition -> Click Workflow Instance -> Click Workflow Instance Name -> Double-click Task Node -> View Task Execution Log`
+![view-log](../../../../img/start/view-log.gif)
 
-![log](../../../../img/new_ui/dev/quick-start/log.png)
+You just finished the first tutorial of DolphinScheduler, you can now run some simple workflow
```
Comment (EricGao888) [inline docs/docs/en/guide/start/quick-start.md]:
```suggestion
You just finished the first tutorial of DolphinScheduler, you can now run some simple workflows
```

**Review by EricGao888 — DISMISSED** [review]
LGTM, except few nits.

File: docs/docs/en/guide/start/quick-start.md [hunk 14]
```diff
@@ -1,61 +1,118 @@
...
+is `admin/dolphinscheduler123`.
+
+![login](../../../../img/start/login.gif)
+
+#### Create Tenant
+
+Tenant is an important concept while using DolphinScheduler, so
+let's briefly introduce the concept of tenant first.
+
+The account named `admin` logged into DolphinScheduler is called user in dolphinscheduler.
```
Comment (zhongjiajie) [inline docs/docs/en/guide/start/quick-start.md]:
```suggestion
DolphinScheduler maps the `admin` account you use to log into DolphinScheduler to `user`.
```

File: docs/docs/en/guide/start/quick-start.md (hunk 5 above)
Comment (zhongjiajie) [reply docs/docs/en/guide/start/quick-start.md]: Add in https://github.com/apache/dolphinscheduler/pull/13607#discussion_r1119528273

**Review by EricGao888 — APPROVED** [review]

**Comment by zhongjiajie** [comment]
Thanks

**Pull request merged**